jenkins-scripts/lib/check_graphic_card.bash

```diff
@@ -3,16 +3,19 @@ GRAPHIC_CARD_PKG=""
 
 export_display_variable()
 {
-    # Hack to found the current display (if available) two steps:
-    # Check for /tmp/.X11-unix/ socket and check if the process is running
-    for i in $(ls /tmp/.X11-unix/ | sed -e 's@^X@:@')
+    # Check for an active X11 display socket.
+    for i in $(find /tmp/.X11-unix -type s)
```

Comment:
Using `find` instead of `ls` makes it possible to print out specifically sockets within the directory rather than all files.